schemas/log.yml

```diff
@@ -156,3 +156,50 @@
 
         According to RFCs 5424 and 3164, the priority is 8 * facility + severity.
         This number is therefore expected to contain a value between 0 and 191.
+
+    - name: syslog.version
+      level: extended
+      type: long
+      example: 1
+      short: Syslog protocol version.
+      description: >
+        The version of the Syslog protocol specification. Only applicable for RFC 5424 messages.
+
+    - name: syslog.hostname
+      level: extended
+      type: keyword
+      example: example-host
+      short: The host that originated the Syslog message.
+      description: >
+        The hostname, FQDN, or IP of the machine that originally sent the Syslog message.
```

Comment:
Nit: I see value in adding a sentence or two here describing how this is the `hostname` taken from the syslog header, and it often can be different a different value from the host that the event took place on.

Won't be an issue for something programmatically parsing like the syslog processor, but it might help clarify confusion for anyone parsing syslog headers themselves.